blueprints/app/files/README.md

```diff
@@ -31,8 +31,8 @@ Make use of the many generators for code, try `ember help generate` for more det
 
 ### Running Tests
 
-* `ember test`
-* `ember test --server`
+* `<% if (yarn) { %>yarn test<% } else { %>npm test<% } %>`
+* `<% if (yarn) { %>yarn test --server<% } else { %>npm test -- --server<% } %>`
```

Comment:
Right, I do like 1 SPOT to look for all errors in all files, continuously. Today, that's the launched Testem UI. It reminds me of things my editor doesn't, because I switched away from the file with errors.

Running another watch process is an option, sure.

Maybe a commit hook would cover my edge case.

Reply:
I don't think a commit hook would solve your live updating needs, that's like a "final check once you're all done", right? not a "while you work" solution.